tests/test_helper.py

```diff
@@ -57,3 +58,11 @@ def test_get_loop_id(sample_orcid_data):
 
 def test_render_orcid_runs():
     render_orcid_qs("0000-0003-4423-4370")
+
+
+def test_get_github():
+    """Test getting a github link."""
+    orcid = "0000-0002-0791-1347"  #  Selja Seppälä
+    data = get_orcid_data(orcid)
+    ids = get_external_ids(data)
+    assert ids["github"] == "seljaseppala"
```

Comment:
at the moment this returns just 's' when running `pyorcidator import --orcid 0000-0002-0791-1347`

Reply:
yes also related to the bug you mentioned, which I believe should be fixed now. Are you on the most recent commit? Note that the test is passing on CI now.